tests/devices/test_default_qubit_tf.py

```diff
@@ -320,6 +320,24 @@ def test_two_qubit_parameters(self, init_state, op, func, theta, tol):
         expected = func(theta) @ state
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
+    def test_apply_ops_dict_above_8_wires(self):
+        """Test that the apply_ops dictionary is empty when more than 8 wires are used"""
+        dev = DefaultQubitTF(wires=9)
+        assert dev._apply_ops == {}
+
+    @pytest.mark.xfail(
+        raises=tf.errors.UnimplementedError,

```

Comment:
You could set this `xfail` to be strict if you want. This will cause the test to fail if it unexpectedly passes (the default behaviour is not to do this).

Reply:
Nice, done!